src/braket/parametric/free_parameter.py

```diff
@@ -99,6 +105,22 @@ def to_dict(self) -> dict:
             "name": self.name,
         }
 
+    @staticmethod
+    def _validate_name(name: str) -> None:
+        if not name:
+            raise ValueError("Symbol names must be non empty")
```

Comment:
```suggestion
            raise ValueError("FreeParameter names must be non empty")
```
also applies to L113, L116

Reply:
Good call.